server/core/storage.go

```diff
@@ -49,12 +50,15 @@ type Storage struct {
 	kv.Base
 	regionStorage    *RegionStorage
 	useRegionStorage int32
+	regionLoaded     bool
+	mu               sync.RWMutex
```

Comment:
How about use atomic? Is it only protect `regionLoaded`?

Reply:
yes,only protect regionLoaded.

Reply:
@nolouch PTAL.thanks